src/EVM/SymExec.hs

```diff
@@ -719,42 +795,43 @@ verifyInputs
   -> Fetch.Fetcher Symbolic m RealWorld
   -> VM Symbolic RealWorld
   -> Postcondition RealWorld
-  -> m (Expr End, [(SMTResult, Expr End)], [(PartialExec, Expr End)])
+  -> m ([(SMTResult, Expr End)], [(PartialExec, Expr End)])
 verifyInputs solvers opts fetcher preState post = do
   conf <- readConfig
   let call = mconcat ["prefix 0x", getCallPrefix preState.state.calldata]
   when conf.debug $ liftIO $ putStrLn $ "   Exploring call " <> call
 
-  expr <- executeVM fetcher opts.iterConf preState
-  when conf.dumpExprs $ liftIO $ T.writeFile "unsimplified.expr" (formatExpr expr)
-  let flattened = flattenExpr expr
-  when (conf.dumpExprs && conf.simp) $ liftIO $ do
-    let exprSimplified = Expr.simplify expr
-    T.writeFile "simplified.expr" (formatExpr exprSimplified)
-    T.writeFile "simplified-conc.expr" (formatExpr $ Expr.simplify $ mapExpr Expr.concKeccakOnePass exprSimplified)
+  ends <- executeVM fetcher opts.iterConf preState
+  -- when conf.dumpExprs $ liftIO $ T.writeFile "unsimplified.expr" (formatExpr expr)
+  -- when (conf.dumpExprs && conf.simp) $ liftIO $ do
+  --   let exprSimplified = Expr.simplify expr
+  --   T.writeFile "simplified.expr" (formatExpr exprSimplified)
+  --   T.writeFile "simplified-conc.expr" (formatExpr $ Expr.simplify $ mapExpr Expr.concKeccakOnePass exprSimplified)
```

Comment:
Probably just delete this?

Reply:
Yep, sorry, fixed. Will push once out of the airport :)